site/docs/v1/tech/installation-guide/monitor.adoc

```diff
@@ -26,9 +26,47 @@ There are a few different APIs exposing FusionAuth application specific informat
 
 In general these are APIs you will have poll to ingest.
 
+=== Log File Formats
+
+.Log File Information
+[cols="1,3,3,3,1"]
+|===
+| Log Type | Export Format | Timezone | Date Format | API Link
+
+| Audit Logs
+| Zipped CSV
+| Specified by the `zoneId` parameter
+| Specified by the `dateTimeSecondsFormat` parameter
+| link:/docs/v1/tech/apis/audit-logs/[API]
+
+| Event Logs
+| JSON
+| UTC
+| link:/docs/v1/tech/reference/data-types/#instants[Instant]
+| link:/docs/v1/tech/apis/event-logs/[API]
+
+| Login Records
+| Zipped CSV
+| Specified by the `zoneId` parameter
+| Specified by the `dateTimeSecondsFormat` parameter
```

Comment:
Defaults to `M/d/yyyy hh:mm:ss a z` ( https://github.com/FusionAuth/fusionauth-app/blob/0c9055eea1ca44bfa80ae2150a776d47eb8f3eb2/src/main/java/io/fusionauth/app/action/api/BaseExportAPIAction.java#L41 )

Same for any action that extends `BaseExportAPIAction`